blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/credential/CredentialContainer.java

```diff
@@ -71,9 +75,14 @@ public CredentialApi get(String name) {
                 }
             }
         }else{
-            for(CredentialsStore store: CredentialsProvider.lookupStores(Jenkins.getInstance())){
-                if(store.getStoreAction() != null) {
-                    apis.add(new CredentialApi(store.getStoreAction(), this));
+            Iterator<BlueOrganization> iterator = OrganizationFactory.getInstance().list().iterator();
```

Comment:
this should actually use the organisation that was part of the URL but there was no `getParent()` here in order to locate the org from what I could see.